microservice-event-trigger/README.md

```diff
@@ -0,0 +1,115 @@
+# Description
+
+This is a self-contained guide that demonstrates setting up an event trigger for a microservice deployed in Kyma. It serves as a refrence for developers who wish to configure their microservices with event trigger.
+
+It uses:
+
+* [http-db-service](https://github.com/kyma-project/examples/tree/master/http-db-service) as a reference microservice that is deployed in the `demo-event-service-trigger` namespace. It will be configured to consume `order.created` event from the application.
+* [commerce-mock](https://github.com/SAP/xf-application-mocks/tree/master/commerce-mock) as a reference enterprise application that connects to Kyma. This application will send the event to Kyma which will trigger the microservice.
+
+### Set up commerce mock.
+
+This section will
+
+* Deploy a mock commerce application on Kyma.
+
+* Expose the commerce mock URL.
+
+* Establish secure connection via the Kyma application connector.
+
+* Register the events that can be sent from the mock commerce application to Kyma.
+
+> **NOTE:** This will be replaced by an actual enterprise application that will connect to Kyma.
+
+1. Create a namespace `mocks` in the Kyma console.
+
+    ![](./assets/create-ns.png)
+
+1. Deploy the [mock commerce application](https://raw.githubusercontent.com/SAP/xf-application-mocks/master/commerce-mock/deployment/xf.yaml).
+
+    ![](./assets/deplo-commerc-mock.png)
+
+1. Create a new application `sample-enterprise`.
+
+    ![](./assets/create-application.png)
+
+1. Copy the URL to connect the application.
+
+    ![](./assets/connect-application.png)
+
+1. Access the commerce mock at https://commerce.{CLUSTER_DOMAIN} .
+
+In the commerce mock User Interface,
+
+1. Connect the `sample-enterprise` to mock commerce.
+
+    ![](./assets/connect-mock-app.png)
+    
+1. Register the `SAP Commerce Cloud - Events`. After registration, it will be available under the `Remote APIs` section.
+
+    ![](./assets/register-events.png)
+
+### Expose the events from the application to namespace in Kyma.
+
+After the application has registered the events that it can send to Kyma, next step would be make them accessible to various serverless workloads (lambdas/microservices) running inside Kyma. Since multiple applications can connect to Kyma, each Kyma namespace is declaratively bound to an application and then the events are made accessible by adding the event catalog to the namespace.
```

Comment:
```suggestion
After the Application registered the Events that it can send to Kyma, you need to make them accessible to various serverless workloads running inside Kyma, such as lambdas and microservices. Since multiple applications can connect to Kyma, each Kyma Namespace is declaratively bound to an Application and then the Events are made accessible by adding the Event catalog to the Namespace.
```